mkt/developers/utils.py

```diff
@@ -154,3 +154,14 @@ def escalate_prerelease_permissions(app, validation, version):
         escalate_app(
             app, version, nobody, 'App uses prerelease permissions',
             mkt.LOG.ESCALATION_PRERELEASE_APP)
+
+
+def prioritize_app(app, user):
+    if not app.priority_review:
```

Comment:
Do you still want to send the comms and log it if the app is already flagged for priority review? Seems like it would make more sense to return here with `if app.priority_review: return`. It would simplify the form also because you could skip that extra check and call this whenever requested.